tools/marvin/marvin/lib/base.py

```diff
@@ -1086,7 +1092,7 @@ def create(cls, apiclient, services, volumeid=None,
     @classmethod
     def register(cls, apiclient, services, zoneid=None,
                  account=None, domainid=None, hypervisor=None,
-                 projectid=None, details=None):
```

Comment:
Any reason to remove the existing parameter? Are you sure it won't affect any of the existing test cases?